esp/scm/ws_workunits.ecm

```diff
@@ -507,6 +507,7 @@ ESPrequest [nil_remove] WUQueryRequest
     [min_ver("1.57")] ESParray<ESPstruct ApplicationValue> ApplicationValues;
     [min_ver("1.72")] string BeforeWU;
     [min_ver("1.72")] string AfterWU;
+    [min_ver("1.77")] unsigned TotalClusterTimeThresholdMilliSeconds(0);
```

Comment:
It's interesting I see 4 examples of "milliseconds" in this file:
TimeoutMilliSec
ElapsedTimeMs
TotalClusterTimeThresholdMilliSeconds
MS
I wonder if "TotalClusterTimeThresholdMs" is obvious enough?  If not, could standardize on the other one that already exists "TotalClusterTimeThresholdMilliSec"?